### ISSUE TITLE
Consider deleting this repo?

### ISSUE THREAD

**Issue by elray1** [hubverse-org/flusight1_hub#6]
My understanding is that this repo is "unofficial", and the [flusight_hub_archive](https://github.com/hubverse-org/flusight_hub_archive) is the more official source?  In that case, should we just delete this repo?

**Comment by bsweger** [comment]
@elray1 Agree that keeping it around is confusing (it no longer syncs to the cloud, and as you note is no longer the official source).

As a concession to those of us with data hoarding tendencies, could we make it a private archived repo _just in case_?

**Comment by nickreich** [comment]
I looked back at this to try to reconstruct what happened. Based on the Network graph today, it looks like we have the flusight1_hub repo as the lower purple fork/line and the flusight_hub_archive as the top blue fork/line. 

![Image](https://github.com/user-attachments/assets/d8b1accb-6dd3-4db1-be50-103c06cc8cab)

I was worried at first that we had done all this work on flusight1_hub that had not been merged back in, but I actually think all of those commits on the purple line were squashed and merged on June 21 in [this commit](https://github.com/hubverse-org/flusight_hub_archive/commit/7fc84cfc1683e9c6538211fac4f39074f269fe5e). As the list of those commit notes is virtually identical to [the list of recent commits to the flusight1_hub repo](https://github.com/hubverse-org/flusight1_hub/commits/main/).

So, I think it actually would be ok to delete.

**Comment by nickreich** [comment]
@bsweger Does this make you less squeamish about a delete? If not, then let's archive it. If so, let's delete it.

**Comment by bsweger** [comment]
> @bsweger Does this make you less squeamish about a delete? If not, then let's archive it. If so, let's delete it.

Nah, don't mind me, I have a repo hoarding problem, and I'm sure it's fine to delete.

**Comment by bsweger** [comment]
@nickreich though I'm assigned to this, I don't have permissions to delete the repo!
